pom.xml

```diff
@@ -401,11 +411,29 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
         <configuration>
           <source>${java.compile.version}</source>
           <target>${java.compile.version}</target>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jarsigner-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>sign</id>
+            <goals>
+              <goal>sign</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <archive>target/dataloader-${project.version}-uber.jar</archive>
+          <skip>${jarsigner.skip}</skip>
```

Comment:
Looking at line 29, the variable is set to 'false' then will it not sign the Jar? Also, how will this work for users who would want to build it on their machine without actually signing?

Reply:
The variable is set to true in pom.xml, thereby skipping jar signing by default. So, anyone who wants to build without signing should run "mvn clean package -DskipTests".